discord/bot.go

```diff
@@ -3,58 +3,70 @@ package discord
 import (
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.